editoast/src/main.rs

```diff
@@ -295,6 +296,30 @@ async fn generate(
     Ok(())
 }
 
+async fn import_rolling_stock(
+    args: ImportRollingStockArgs,
+    pg_config: PostgresConfig,
+) -> Result<(), Box<dyn Error + Send + Sync>> {
+    let rolling_stock_file = File::open(args.rolling_stock_path)?;
+    let manager = ConnectionManager::<PgConnection>::new(pg_config.url());
+    let pool = Data::new(Pool::builder(manager).build().unwrap());
+    let mut rolling_stock: RollingStockModel =
+        serde_json::from_reader(BufReader::new(rolling_stock_file))?;
+    info!(
+        "🍞 Importing rolling stock {}",
+        rolling_stock.name.clone().unwrap().bold()
+    );
+    rolling_stock.locked = Some(true);
```

Comment:
The imported rolling stocks shouldn't be locked.

```suggestion
    rolling_stock.locked = Some(false);
```